src/client/Client.cc

```diff
@@ -11258,6 +11258,9 @@ int Client::statfs(const char *path, struct statvfs *stbuf,
   // enabled
   ceph_assert(cct->_conf.get_val<bool>("client_quota") == false || quota_root != nullptr);
 
+  /* If bytes quota is set on a directory and conf option "client quota df"
```

Comment:
@rishabh-d-dave You mentioned that the comment was added since it was hard to find where this is implemented. IMO, file system stats (free, used, etc..) are implemented in statfs, so I'm not sure what was hard to find.